src/VerticalTabs/index.tsx

```diff
@@ -220,4 +214,4 @@ const output: typeof VerticalTabs & {
   return output;
 })();
 
-export default output;
+output;
```

Comment:
No export / return?

Reply:
fixed this in my current branch, yes needs an export